ftw/footer/footer_viewlet.py

```diff
@@ -1,5 +1,7 @@
 from AccessControl import getSecurityManager
 from Acquisition._Acquisition import aq_parent
+
```

Comment:
Why is there an extra newline? And did you sort imports here?

Reply:
Maybe thumb felt heavy? ;-)   No, it's ok tho, no?